src/views/v2/commerce/checkout/anonymous.php

```diff
@@ -0,0 +1,33 @@
+<?php
+/**
+ * Tickets Commerce: Checkout Page Header
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/tickets/v2/commerce/checkout/anonymous.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link    https://evnt.is/1amp Help article for RSVP & Ticket template files.
+ *
+ * @since   TBD
+ *
+ * @version TBD
+ *
+ * @var \Tribe__Template $this                  [Global] Template object.
+ * @var bool $anonymous [Global] User state.
```

Comment:
Maybe use `$is_anonymous` so it's clearer and more in line with what we call other variables. Can't we get this with `must_login` and the user not logged in? If not, we need to add more information to the variable docblock.